bibliopixel/project/types/ledtype.py

```diff
@@ -0,0 +1,16 @@
+import functools
+from ...drivers.serial import codes
+
+USAGE = """An LEDTYPE is represented by a string.
```

Comment:
Ah yeah... I forgot in that last PR I wanted to consolidate this with SPI_LEDTYPE: https://github.com/ManiacalLabs/BiblioPixel/blob/master/bibliopixel/drivers/SPI/driver.py#L13
I can make them the same easily. But I'll leave that for another future PR.